.env.example

```
@@ -0,0 +1,14 @@
+POSTGRES_PASSWORD=
+POSTGRES_DB=
+
+# JWT
+JWT_SECRET=
+
+# Kafka
+KAFKA_BOOTSTRAP_SERVERS=
+
+# Redis
+REDIS_HOST=
+REDIS_PORT=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)